src/AppInstallerCLICore/Commands/InstallCommand.cpp

```diff
@@ -143,12 +145,28 @@ namespace AppInstaller::CLI
     {
         context.SetFlags(ContextFlag::ShowSearchResultsOnPartialFailure);
 
-        context <<
-            Workflow::ReportExecutionStage(ExecutionStage::Discovery) <<
-            Workflow::GetManifest <<
-            Workflow::SelectInstaller <<
-            Workflow::EnsureApplicableInstaller <<
-            Workflow::CheckForUnsupportedArgs <<
-            Workflow::InstallSinglePackage;
+        if (context.Args.Contains(Execution::Args::Type::Manifest))

```

Comment:
side note: I've thought that it would be great to collapse the manifest flow into the normal one by making `OpenSource` create a simple in-memory source for the manifest so that it could behave exactly as everything else.